tools/cgeist/driver.cc

```diff
@@ -239,6 +254,81 @@ static int ExecuteCC1Tool(SmallVectorImpl<const char *> &ArgV) {
   return 1;
 }
 
+bool getCudaPaths(std::string &libDevicePath, std::string &ptxasPath) {
```

Comment:
@wsmoses Getting the paths properly from the driver/toolchain classes seems too hard because functionality is hidden behind private fields/methods so this is what I opted for